docs/tempo/website/operations/polling.md

```diff
@@ -3,7 +3,7 @@ title: Polling and monitoring
 weight: 8
 ---
 
-# Polling
+# Polling and monitoring
```

Comment:
The associated section titled Monitoring is specific to Polling. To me, adding Monitoring to the title makes it sound like the page covers Polling and Monitoring topics completely. Would prefer leaving this one as is, maybe we can change `title` to `Polling` as well